Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorState.cs

```diff
@@ -66,6 +66,17 @@ public InputActionsEditorState(InputActionsEditorState other, SerializedObject a
             m_selectedControlSchemeIndex = other.m_selectedControlSchemeIndex;
             m_selectedDeviceRequirementIndex = other.m_selectedDeviceRequirementIndex;
 
+            // Selected ControlScheme index is serialized but we have to recreated actual object after domain reload
+            if (m_selectedControlSchemeIndex != -1)
+            {
+                var controlSchemeSerializedProperty = serializedObject
+                    .FindProperty(nameof(InputActionAsset.m_ControlSchemes))
+                    .GetArrayElementAtIndex(m_selectedControlSchemeIndex);
+
+                m_ControlScheme = new InputControlScheme(controlSchemeSerializedProperty);
+            }
+            else m_ControlScheme = new InputControlScheme();
```

Comment:
Minor but would recommend keeping consistent if/else style here for readability and avoiding future mistakes so suggest keeping else clause also inside brackets and with line break similar to the if above.

Reply:
Oh yeah, that's one of my own personal styling quirks.
I tend to prefer having single-line else blocks on the same line as the else, but I'll be sure to use the correct package styling (and will fix this in next PR).